Zend/zend_language_scanner.l

```diff
@@ -1079,6 +1080,12 @@ static void zend_scan_escape_string(zval *zendlval, char *str, int len, char quo
 
 int lex_scan(zval *zendlval)
 {
+
+static unsigned int in_class = 0;
+static zend_bool    in_const_list = 0;
+
+ZEND_ASSERT(in_class >= 0);
```

Comment:
I removed the unsigned and kept the assertion because it's currently not very obvious to detect bugs if somebody forget to increment/decrement of `in_class`.